src/newLaunch/seed/stages/stage4.html

```diff
@@ -120,6 +120,22 @@
               </div>
             </div>
           </div>
+          <div class="infoInput csvList">
+              <div class="csvUploadInputContainer">
+              <label for="allowList"><span class="csvLabelContent">Allowlist (Optional)</span><question-mark text="Upload your allowlist in CSV format"></question-mark></label>
+              <span class="fakeLabel">
+                <input class="footer-input"
+                id="allowList"
+                type="file"
+                accept=".csv"
+                files.bind="csv"
+                placeholder="Upload your .csv allowlist..." />
+              </div>
+            </span>
+              <div >
+                <button class="upload-csv" click.delegate="parseCsv()">Upload</button>
```

Comment:
I expected to have have the file upload popup appear when pressing on "Upload".
(currently popup appears when clicking on the input)

Can we have an UX like this
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/file

I added 2 Tasks to the ticket
https://app.shortcut.com/curvelabs/story/300/add-csv-upload-input-to-seed-details-stage-4

Reply:
same mdn article displays this pattern, which is closer to our behaviour (minus the browser button):
![image](https://user-images.githubusercontent.com/85691163/190573776-0ae42ddb-7020-42ff-8c2c-57c582170305.png)